include/common_structs.h

```diff
@@ -102,7 +102,12 @@ typedef struct BlurBuffer {
     /* 0xA4 */ f32 z[20];
 } BlurBuffer; // size = 0xF4
 
-typedef s32 Palette16[16]; // size = 0x40
+typedef s16 Palette16[16]; // size = 0x20
+
+typedef union {
+    u16 h;
+    u32 w;
+} test;
```

Comment:
feedback on the name would be appreciated

Reply:
Might be better to not typedef it for now and just do `union { ... } currentAnim` on L123

Reply:
Also what's the reasoning for naming the fields `h` and `w`?

Reply:
sounds like a good idea to just do an anonymous union. As for the field names; "halfword" and "word" are the idea behind them

Reply:
oh I was thinking height and width. I think we might want to go with just u32 and u16 or something